patchwork/datastore.py

```diff
@@ -132,6 +135,21 @@ def register_promisee(self, address: Address, promisee: Any) -> None:
         else:
             raise ValueError("address not a promise")
 
+    # Note: This doesn't look elegant, but it's better than ``return
+    # self.promises.get(address) …``, which would silently ignore wrong
+    # addresses.
```

Comment:
Probably fine to leave out the comment - I'm not even sure there really is a functionally equivalent alternative that looks substantially more elegant, even apart from the KeyError consideration.

Reply:
Looking at the code again, I agree. I'll remove the comment.

Reply:
Done.